.goreleaser-nightly.yml

```diff
@@ -1,7 +1,7 @@
 project_name: nats-server
 
 builds:
-  - main: ./main.go
+  - main: .
```

Comment:
We know this works yes?

Reply:
Yes, this is what is used in `.goreleaser.yml` but was overlooked for the `-nightly` variant when the fix was introduced last release (affects vulnerability scanners).

Reply:
Confirmed this should generate the correct metadata:

```
go version -m dist/nats-server_linux_amd64_v1/nats-server
dist/nats-server_linux_amd64_v1/nats-server: go1.19.5
	path	github.com/nats-io/nats-server/v2
	mod	github.com/nats-io/nats-server/v2	(devel)	
	dep	github.com/klauspost/compress	v1.16.5	h1:IFV2oUNUzZaz+XyusxpLzpzS8Pt5rh0Z16For/djlyI=
	dep	github.com/minio/highwayhash	v1.0.2	h1:Aak5U0nElisjDCfPSG79Tgzkn2gl66NxOMspRrKnA/g=
	dep	github.com/nats-io/jwt/v2	v2.4.1	h1:Y35W1dgbbz2SQUYDPCaclXcuqleVmpbRa7646Jf2EX4=
	dep	github.com/nats-io/nkeys	v0.4.4	h1:xvBJ8d69TznjcQl9t6//Q5xXuVhyYiSos6RPtvQNTwA=
	dep	github.com/nats-io/nuid	v1.0.1	h1:5iA8DT8V7q8WK2EScv2padNa/rTESc1KdnPw4TC2paw=
	dep	go.uber.org/automaxprocs	v1.5.1	h1:e1YG66Lrk73dn4qhg8WFSvhF0JuFQF0ERIp4rpuV8Qk=
	dep	golang.org/x/crypto	v0.8.0	h1:pd9TJtTueMTVQXzk8E2XESSMQDj/U7OUu0PqJqPXQjQ=
	dep	golang.org/x/sys	v0.7.0	h1:3jlCCIQZPdOYu1h8BkNvLz8Kgwtae2cagcG/VamtZRU=
	dep	golang.org/x/time	v0.3.0	h1:rg5rLMjNzMS1RkNLzCG38eapWhnYLFYXDXj2gOlr8j4=
	build	-compiler=gc
	build	-ldflags="-w -X github.com/nats-io/nats-server/v2/server.gitCommit=f3dac91d"
	build	CGO_ENABLED=0
	build	GOARCH=amd64
	build	GOOS=linux
	build	GOAMD64=v1
	build	vcs=git
	build	vcs.revision=f3dac91d2a0b0503e0438d3dc2e5c6161a5b65a3
	build	vcs.time=2023-05-18T17:57:40Z
	build	vcs.modified=true
```

Also trivy / grype vulnerability checks are fine so current deps are ok.